src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueWallScreen.java

```diff
@@ -684,6 +687,7 @@ private void resetAllInstances() {
             this.resetInstance(instance, false, false, playSound);
         }
         this.blockedMainPositions.clear();
+        this.showFinishedBenchmarkResults = false;
```

Comment:
This shouldn't be necessary since `resetAllInstances` calls `resetInstance` which already resets the field.
Only exception would be if there is no instance to be reset, but I don't think that has to be covered here.
Same thing goes for `focusReset` and `resetRow`.

Reply:
> This shouldn't be necessary since `resetAllInstances` calls `resetInstance` which already resets the field. Only exception would be if there is no instance to be reset, but I don't think that has to be covered here. Same thing goes for `focusReset` and `resetRow`.

ok i'll remove it from here